src/gui/folderwatcher.cpp

```diff
@@ -57,7 +58,7 @@ bool FolderWatcher::pathIsIgnored( const QString& path )
 
 #ifndef OWNCLOUD_TEST
     if (_folder->isFileExcludedAbsolute(path)) {
-        qDebug() << "* Ignoring file" << path;
+        qCDebug(lcFolderWatcher) << "* Ignoring file" << path;
```

Comment:
It's useful when debugging folder watcher issues, but I'm okay with it being in debug.